script/Arcueid/Day 4/04_00_ARC04_3.txt

```diff
@@ -375,14 +375,14 @@ No. It's just a coincidence. It %{i}has%{/i} to be a coincidence.
 -- Page 120, Offset 5882.
 -- 感触がない。関連がない。関係がない。
 // 感触 is explicitly _tactile_
-I've never come into contact with this woman.
- I have no connection to her.
- She has nothing to do with me.
+We have never touched.
+ We have no relation.
+ We are complete strangers.
```

Comment:
Relation one of those words that always feels awkward to me in contexts like this, maybe
```
We've never touched. 
 We've never met. 
 We're complete strangers.
```